content/workshops/_index.md

```diff
@@ -31,7 +31,7 @@ The Carpentries teaches workshops around the world on the foundational skills to
 </table>
 {{< /table >}}
 
-[Request a workshop]({{< param amy_workshop_landing >}})
+[Host a workshop]({{< param amy_host-workshop_landing >}})
```

Comment:
I tried and messed up the code, but I would like the button to direct to the host a workshop landing.